### ISSUE TITLE
Investigate how to append text for screen readers

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#482]
**Is your feature request related to a problem? Please describe.**
We need to append some text to the caption, #464.

**Describe the solution you'd like**
An investigation into the use of `aria-labelledby` and any alternative solutions. This investigation will inform an implementation.

**Additional Context**
See the comments in #464.

**Comment by greenc05** [comment]
I've looked into the implementation suggested by @sareh https://dequeuniversity.com/presentations/html5-nfb/figure-figcaption and this isn't appropriate for our use case. 

We need alt text to describe the image and we also need a caption, which can provide context for the image and how it relates to the story, it should not describe the image. In @sareh's suggestion, the caption is read out in place of the image alt text, which does not describe the image and thus this approach does not meet our needs. Please follow the solution described in https://github.com/bbc/simorgh/issues/464

**Comment by pjlee11** [comment]
@greenc05 do you think there will be other use cases to utilise `aria-labelledby`? This issue was prompted by the image work, however it is also an investigation in how we would apply text for screen readers for the entire application.

**Comment by greenc05** [comment]
@pjlee11 there will no doubt be other use cases where we can use aria, such as aria-labelledby, I have been carrying out best practice implementations for all the screen reader UX and will endeavour to pass this information over to the team when it's not conveyed in the UX doc.
